Tests/Parser/fixtures/oss.yml

```diff
@@ -4961,11 +4961,11 @@
 -
   user_agent: NetworkingExtension/8619.1.10.2 CFNetwork/1555.1 Darwin/24.0.0
   os:
-    name: iOS
-    short_name: IOS
-    version: "18.0"
+    name: Mac
+    short_name: MAC
+    version: "15.0"
```

Comment:
https://theapplewiki.com/wiki/Kernel#macOS/OS_X

Darwin/24.0.0  - 15.1
Darwin/24.2.0  - 15.2

Reply:
Darwin/24.0.0 - 15.1 - this is 15.0 or 15.0.1

![Untitled](https://github.com/user-attachments/assets/7906f1c5-cbac-4b6c-a428-2cf786546776)

Darwin/24.2.0 - 15.2 - this is already detected
https://github.com/matomo-org/device-detector/blob/05a87ab362056c186b66c2266dcc73cae5812e23/regexes/oss.yml#L1602-L1603